CLAUDE.md

```
@@ -169,17 +189,22 @@ uv run pytest -x                     # Fail fast on first error
 
 ### Scripts
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)